### ISSUE TITLE
Why is umbrel connecting to something on clearnet which will reveal my real IP address?

### ISSUE THREAD

**Issue by gitotilla** [getumbrel/umbrel#1680]
https://github.com/getumbrel/umbrel/blob/5fb00dbe1769f0c5142276dc16cad3774d23003c/packages/manager/logic/system.js#L84

**Comment by gitotilla** [comment]
https://api.umbrel.com is a clearnet address.  Wouldn't my node connecting to that address reveal my real IP address?

**Comment by gitotilla** [comment]
https://api.umbrel.com/ is a clearnet address. Wouldn't my node connecting to that address reveal my real IP address?

See https://twitter.com/runcitadel/status/1680227793106247681 for discussion thread on this.

**Comment by warioishere** [comment]
they want to track you, privacy is not their goal

**Comment by highghlow** [comment]
> https://api.umbrel.com/ is a clearnet address. Wouldn't my node connecting to that address reveal my real IP address?

Yes, it does. This url is used for update checking and the solution would be to add a toggle: "Tunnel updates through TOR"
